validator/accounts/accounts.go

```diff
@@ -0,0 +1,13 @@
+package accounts
+
+import (
+	"github.com/prysmaticlabs/prysm/validator/accounts/wallet"
+	"github.com/prysmaticlabs/prysm/validator/keymanager"
+)
+
+// AccountConfig specifies parameters to run to delete, enable, disable accounts.
+type AccountConfig struct {
```

Comment:
This might be overkill.

Happy to create an ActivationAccountConfig if we want to keep each config distinct.

Reply:
This is OK, just concerned what the PublicKeys [][]byte are for, as it is not clear what to use them for. Could we specify `DeletePublicKeys` and `EnablePublicKeys` and `DisablePublicKeys` ? Better to repeat ourselves and be more explicit with these config structs

Reply:
Also rename to AccountsConfig

Reply:
Just to be clear @rauljordan, are you suggesting:

```
type AccountsConfig struct {
        Wallet     *wallet.Wallet
	Keymanager keymanager.IKeymanager
	DeletePublicKeys [][]byte
        EnablePublicKeys [][]byte
        DisablePublicKeys [][]byte
}
```

Reply:
Yep